docs/CHANGELOG.asciidoc

```diff
@@ -40,6 +40,15 @@ This section summarizes the changes in each release.
 
 See <<breaking-changes-7.6,breaking changes in 7.6>>.
 
+[float]
+[[known-issue-7.6.2]]
+=== Known issue
+
+Configuration::
+* The `server.customResponseHeaders` option will prevent {kib} to start if any headers are set using other types than string.
+You can convert your boolean or number headers to strings to fix this. (`my-header: "true"` instead of `my-header: true`).
```

Comment:
```suggestion
To fix this, convert your boolean and number headers to strings.  For example, use`my-header: "true"` instead of `my-header: true`.
```